OpenRA.Mods.Common/UpdateRules/Rules/DefineLocomotors.cs

```diff
@@ -0,0 +1,162 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class DefineLocomotors : UpdateRule
+	{
+		public override string Name { get { return "Introduce global Locomotor definitions"; } }
+		public override string Description
+		{
+			get
+			{
+				return "A large number of properties have been moved from the actor-level Mobile trait\n" +
+					"to either world-level Locomotor traits or new actor-level GrantCondition* traits.";
```

Comment:
Can we please add a line explaining what the rule actually does? Something like "Conditions for subterranean and jumpjet behaviours are migrated, and affected Mobile traits are listed for inspection."